source/aks/if-something-goes-wrong.html.md.erb

```diff
@@ -0,0 +1,167 @@
+---
+title: If Something Goes Wrong
+weight: 50
+---
+
+# <%= current_page.data.title %>
+
+
+If you come across a failed AKS resource or you have pushed in breaking changes that have caused failures in AKS, you may find the below guidance helpful in resolving the issues. 
+
+If [upgrading](upgrading-aks-clusters.html) or [re-building](rebuilding-aks-clusters.html) the AKS Clusters most of the times the process will go on without any issues if the steps in the respective guides have been followed thoroughly.
+
+The most common AKS cluster-related issues and resolution steps have been covered in the Troubleshooting page [here](https://hmcts.github.io/ops-runbooks/aks/known-issues.html). If you come across a problem with the AKS Cluster that has not been covered in the page, please add it in, so that we have a reference to it in case the issue happens again in the future.
```

Comment:
references to pages within the same repo should be local.
In this case it should just be `known-issues.html` so that middleman server references the local copy when testing.